extensions/kind/src/extension.ts

```diff
@@ -222,6 +220,33 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   extensionApi.provider.onDidUnregisterContainerConnection(() => {
     searchKindClusters(provider);
   });
+  extensionApi.provider.onDidUpdateProvider(() => registerProvider(extensionContext, provider));
+}
+
+export async function activate(extensionContext: extensionApi.ExtensionContext): Promise<void> {
+  const installer = new KindInstaller(extensionContext.storagePath);
+  kindCli = await detectKind(extensionContext.storagePath, installer);
+
+  if (!kindCli) {
+    if (await installer.isAvailable()) {
+      const statusBarItem = extensionApi.window.createStatusBarItem();
+      statusBarItem.text = 'Kind';
+      statusBarItem.tooltip = 'Kind not found on your system, click to download and install it';
```

Comment:
it's missing probably
```
statusBarItem.iconClass = 'fa fa-exclamation-triangle';
```

to notify that there is a problem on Kind like 
![image](https://user-images.githubusercontent.com/436777/218439367-7c1b2b9b-4d6b-49da-b06b-c4100240910b.png)